src/main/java/store/Application.java

```diff
@@ -0,0 +1,30 @@
+package store;
+
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import store.model.Product;
+import store.repository.ProductRepository;
+
+@SpringBootApplication
+@EntityScan("store.model")
+@EnableJpaRepositories("store.repository")
+public class Application {
+    @Autowired
+    private ProductRepository productRepository;
+    public static void main(String[] args) {
+        SpringApplication.run(Application.class, args);
+    }
+
+    @Bean
+    InitializingBean sendDatabase() {
```

Comment:
please remove test data. If you need new product - use postman
Even if its a sandbox please do it in a right way